packages/andromeda-finance/src/validator_staking.rs

```diff
@@ -0,0 +1,24 @@
+use andromeda_std::{andr_instantiate, andr_instantiate_modules, error::ContractError};
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, DepsMut};
+
+#[andr_instantiate]
+#[andr_instantiate_modules]
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub default_validator: Addr,
+}
+
+impl InstantiateMsg {
+    pub fn validate(&self, deps: &DepsMut) -> Result<bool, ContractError> {
+        validate_validator(deps, &self.default_validator)
+    }
+}
+
+pub fn validate_validator(deps: &DepsMut, validator: &Addr) -> Result<bool, ContractError> {
```

Comment:
This is a bit of an odd name but I'm struggling to think of something better, maybe something like `check_is_validator`?

Reply:
Well, I don't like that name as well. I normally use `is_` prefix for validating functions. I thought of `is_valid_validator` but could not find functions with that convention.

Reply:
Changed function name to `is_validator`